### ISSUE TITLE
Crash with invalid macro

### ISSUE THREAD

**Issue by kirbyfan64** [tonyfischetti/sake#64]
I'd put a test case, but the bug is slightly obvious:

```
Traceback (most recent call last):
  File "/usr/local/bin/sake", line 5, in <module>
    pkg_resources.run_script('master-sake==0.9.6', 'sake')
  File "build/bdist.linux-x86_64/egg/pkg_resources.py", line 528, in run_script
    dist.insert_on(self.entries, entry)
  File "build/bdist.linux-x86_64/egg/pkg_resources.py", line 1401, in run_script
    raise IOError('"os.rename" and "os.unlink" are not supported '
  File "/usr/local/lib/python2.7/dist-packages/master_sake-0.9.6-py2.7.egg/EGG-INFO/scripts/sake", line 128, in <module>
    
  File "build/bdist.linux-x86_64/egg/sakelib/acts.py", line 308, in expand_macros
  File "build/bdist.linux-x86_64/egg/sakelib/acts.py", line 308, in expand_macros
  File "build/bdist.linux-x86_64/egg/sakelib/acts.py", line 297, in expand_macros
NameError: global name 'InvalidMacroError' is not defined
```

**Comment by kirbyfan64** [comment]
FYI, this caused by https://github.com/tonyfischetti/sake/commit/dc89add416a63979e4c5a0ef4ba8af4f675d0e1e.

**Comment by tonyfischetti** [comment]
Fixed by d2d0a4ac19951e3e95c705d3182e915d31ff9293
